test/actions/FilterActionsTest.js

```diff
@@ -3,59 +3,6 @@ import { expect } from 'chai'
 import * as filterActions from '../../src/actions/filterActions'
 
 describe('FilterActions', function () {
-    describe('serializeFilters', function () {
```

Comment:
Looks like this cleanup may have been overzealous. The command `git checkout master -- FILE_NAME` overwrites your local version of the file with the master version, and I think you used it on this file.

Reply:
No, I realized that none of it applied anymore.  I'll add testing and request feedback later today.